frame/uniswap-v2/README.md

```diff
@@ -0,0 +1 @@
+Uniswap-v2 pallet which has constant-product AMM alogrithm as explained in https://hackmd.io/@HaydenAdams/HJ9jLsfTz?type=view
```

Comment:
please add here other papers you have read

Reply:
Uniswap-v2 white paper does not cover much details about implementation. I have refered solidity code of Uniswap-V2. I will add links to that.